content/en/account_management/billing/product_allotments.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Product Allotments
+kind: documentation
+---
+
+Allotments provide additional usage that comes with subscriptions to select parent products. They grant a certain amount of usage for a child product as part of the account's committed and on-demand usage of the parent product.
+
+Examples of products that have this structure include Infrastructure hosts and containers, where every host comes with a container allotment.
+
+## Allotments within billing calculations
+
+Total usage incurred is classified into billable and non-billable usage. Billable usage is what an account can be charged for, while non-billable usage is not charged. Non-billable usage can include trial usage.
+
+In order to invoice billable usage, included usage is first subtracted. Allotments are factored into included usage, which is then used to calculate on-demand usage from billable usage:
+
+- `allotments + committed usage = included usage`
+- `billable usage - included usage = on-demand usage`
+
+For example, an account can have a total Ingested Spans usage of 150 GB. From this, 140 GB is billable usage. If there is a prior commitment of 50 GB and an allotment of 30 GB, this 80 GB of usage is classified as included usage and subtracted from the 140 GB of billable usage. The remaining 60 GB of usage is classified as on-demand usage.
+
+- To view total usage and billable usage, see the **All** and **Billable** tabs within the [**Plan and Usage** page][2] in Datadog. 
+- To view commitments, refer to your contract.
+
+## Calculating included usage
+Total included usage is made up of the **commitment for the product, and the sum of allotments per parent product**. Refer to the user's contract for commitment quantities. The following variables determine how allotment usage is calculated:
+
+- On-demand option
+- Allotments by parent products
+- Usage aggregation function
+
+### On-demand option
+
+The allotment usage of a product can be computed according to the organization's on-demand metering option. Organizations can opt for a **monthly or hourly** on-demand option. Refer to your contract for information on your metering option. 
+
+By default, the on-demand option is set at the subscription level and applies to all products except for the following, which support a single default: 
```

Comment:
Can we rephrase the last portion of this sentence to read, "which support a single default on-demand option"?